### PR TITLE
Dx 1440/server

### PR DESCRIPTION
**Pull request by BryanZWu**

### CODE REVIEW

File: assets/server/node/index.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+const express = require('express');
+const ngrok = require('ngrok');
+const numbers = require('@bandwidth/numbers');
+const readline = require('readline');
+//====================INPUTS==========================
+const port = 80;
```
Comment (jmulford-bw) [inline assets/server/node/index.js:6]: I think 3000 would be a better default port. express typically defaults to port 3000 and some devs may have port 80 already taken up by something like tomcat

File: assets/server/node/README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+## Instructions
+Set the following environment variables:
+```
+set BANDWIDTH_ACCOUNT_ID myAccId
+set BANDWIDTH_API_USER myDashboardUser
+set BANDWIDTH_API_PASSWORD myDashboardPassword
+```
+Note that these are the same credentials you used to log into the CLI.
+
+After this, `npm start` will create the server and update the callback url of the listed application to this server
```
Comment (jchavez443) [inline assets/server/node/README.md:12]: How do I set the `listed application` ?

Comment (BryanZWu) [reply assets/server/node/README.md]: Good catch. I've updated the readme, but essentially in index.js you have 
image and the CLI, when creating the file, automatically replaces that string with the cli-default application.

File: assets/server/node/index.js (hunk 1 above)
Comment (BryanZWu) [reply assets/server/node/index.js]: Sounds good. Fixed

**Pull request merged**